packages/api/db/migration/20221114230005_alter_field_work_task_table.js

```diff
@@ -0,0 +1,97 @@
+// import fieldWorkData from '../../db/seeds/seedData/field_work.json';
+
+export const up = async function (knex) {
+  const fieldWorkData = [
+    {
+      field_work_type_translation_key: 'COVERING_SOIL',
+      field_work_name: 'Covering soil',
+    },
+    {
+      field_work_type_translation_key: 'FENCING',
+      field_work_name: 'Fencing',
+    },
+    {
+      field_work_type_translation_key: 'PREPARING_BEDS_OR_ROWS',
+      field_work_name: 'Preparing beds or rows',
+    },
+    {
+      field_work_type_translation_key: 'PRUNING',
+      field_work_name: 'Pruning',
+    },
+    {
+      field_work_type_translation_key: 'SHADE_CLOTH',
+      field_work_name: 'Shade cloth',
+    },
+    {
+      field_work_type_translation_key: 'TERMINATION',
+      field_work_name: 'Termination',
+    },
+    {
+      field_work_type_translation_key: 'TILLAGE',
+      field_work_name: 'Tillage',
+    },
+    {
+      field_work_type_translation_key: 'WEEDING',
+      field_work_name: 'Weeding',
+    },
+  ];
+  await knex.schema.createTable('field_work', function (t) {
+    t.increments('field_work_id').primary();
+    t.uuid('farm_id').references('farm_id').inTable('farm').defaultTo(null);
+    t.string('field_work_name');
+    t.string('field_work_type_translation_key');
+    t.string('created_by_user_id').references('user_id').inTable('users').defaultTo('1');
+    t.string('updated_by_user_id').references('user_id').inTable('users').defaultTo('1');
+    t.dateTime('created_at').defaultTo(new Date('2000/1/1').toISOString()).notNullable();
+    t.dateTime('updated_at').defaultTo(new Date('2000/1/1').toISOString()).notNullable();
+    t.unique(['field_work_name', 'field_work_type_translation_key']);
+  });
+
+  await knex('field_work').insert(fieldWorkData);
+
+  const field_work_rows = await knex('field_work').select(
+    'field_work_id',
+    'field_work_type_translation_key',
+  );
+
+  await knex.schema.alterTable('field_work_task', (t) => {
```

Comment:
@anant00729 

Is it ok to alter two tables in one migration? What is best practice?

Reply:
in the alter table query I am just adding the columns in the up function. so it's a safe change.